go/vt/vtgate/planbuilder/testdata/dml_cases.txt

```diff
@@ -1172,6 +1172,41 @@
   }
 }
 
+# insert for overlapped vindex columns
+"insert overlap_vindex (kid, column_a, column_b) VALUES (1,2,3)"
+{
+  "Original": "insert overlap_vindex (kid, column_a, column_b) VALUES (1,2,3)",
+  "Instructions": {
+    "Opcode": "InsertSharded",
+    "Keyspace": {
+      "Name": "user",
+      "Sharded": true
+    },
+    "Query": "insert into overlap_vindex(kid, column_a, column_b) values (:_kid0, :_column_a0, 3)",
+    "Values": [
```

Comment:
this would be easier to understand if we called this `VindexValues` while converting to JSON just like in the original `Insert` struct.

Reply:
Yeah. Makes sense. We should make a separate PR.